test_requirements.txt

```diff
@@ -29,6 +29,7 @@ pyarrow>=0.12.0, <1.0.0
 pylint>=2.4.4, <3.0
 pyspark>=2.2.0, <3.0; python_version < '3.8'
 pytest-cov>=2.5, <3.0
+pytest-html>=2.0.1
```

Comment:
Can you upper bound this by 3.0?

Reply:
Sure!

Reply:
Done via f8718aa49d17ae7da5386c47340ff045f81ca6ae